bok_choy/page_object.py

```diff
@@ -282,11 +282,20 @@ def visit(self):
         # A BrokenPromise will be raised if the page object's is_browser_on_page method
         # does not return True before timing out.
         try:
-            return self.wait_for_page()
+            page_wait = self.wait_for_page()
         except BrokenPromise:
             raise PageLoadError("Timed out waiting to load page '{!r}' at URL '{}'".format(
                 self, self.url
             ))
+        
+        assert False, "Making sure this will fail"
+        assert "<XSS" not in self.browser.page_source, "page contains XSS vulnerability"
+        assert "<xss" not in self.browser.page_source, "page contains XSS vulnerability"
+
```

Comment:
both case sensitive cases can be handled in one:
`assert "<xss" not in self.browser.page_source.lower(), "page contains XSS vulnerability"`